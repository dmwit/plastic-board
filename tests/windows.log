This is pdfTeX, Version 3.1415926-2.5-1.40.14 (TeX Live 2013/W32TeX) (format=pdflatex 2014.1.17)  19 JAN 2014 11:04
entering extended mode
 restricted write18 enabled.
 %&-line parsing enabled.
**"D:/Dropbox/temp/test.tex"
! Emergency stop.
<*> "D:/Dropbox/temp/test.tex"

*** (job aborted, file error in nonstop mode)


Here is how much of TeX's memory you used:
 5 strings out of 493308
 173 string characters out of 6141367
 52419 words of memory out of 5000000
 3564 multiletter control sequences out of 15000+600000
 3640 words of font info for 14 fonts, out of 8000000 for 9000
 957 hyphenation exceptions out of 8191
 0i,0n,0p,1b,6s stack positions out of 5000i,500n,10000p,200000b,80000s
!  ==> Fatal error occurred, no output PDF file produced!
