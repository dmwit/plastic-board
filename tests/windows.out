([],((Nothing,Nothing),Boring "This is pdfTeX, Version 3.1415926-2.5-1.40.14 (TeX Live 2013/W32TeX) (format=pdflatex 2014.1.17)  19 JAN 2014 11:04"))
([],((Nothing,Nothing),Boring "entering extended mode"))
([],((Nothing,Nothing),Boring " restricted write18 enabled."))
([],((Nothing,Nothing),Boring " %&-line parsing enabled."))
([],((Nothing,Nothing),Boring "**\"D:/Dropbox/temp/test.tex\""))
([],((Nothing,Nothing),Unknown "! Emergency stop."))
([],((Nothing,Nothing),Unknown "<*> \"D:/Dropbox/temp/test.tex\""))
([],((Nothing,Nothing),Boring ""))
([],((Nothing,Nothing),Unknown "*** (job aborted, file error in nonstop mode)"))
([],((Nothing,Nothing),Boring ""))
([],((Nothing,Nothing),Boring ""))
([],((Nothing,Nothing),Boring "Here is how much of TeX's memory you used:"))
([],((Nothing,Nothing),Boring " 5 strings out of 493308"))
([],((Nothing,Nothing),Boring " 173 string characters out of 6141367"))
([],((Nothing,Nothing),Boring " 52419 words of memory out of 5000000"))
([],((Nothing,Nothing),Boring " 3564 multiletter control sequences out of 15000+600000"))
([],((Nothing,Nothing),Boring " 3640 words of font info for 14 fonts, out of 8000000 for 9000"))
([],((Nothing,Nothing),Boring " 957 hyphenation exceptions out of 8191"))
([],((Nothing,Nothing),Boring " 0i,0n,0p,1b,6s stack positions out of 5000i,500n,10000p,200000b,80000s"))
([],((Nothing,Nothing),Unknown "!  ==> Fatal error occurred, no output PDF file produced!"))
